custom/icds_reports/views.py

```diff
@@ -402,7 +402,7 @@ def get(self, request, *args, **kwargs):
                 for loc in locations:
                     location_key = '%s_id' % loc.location_type.code
                     config.update({
-                        location_key: loc.location_id,
+                        location_key: "d5d0fce5e73ff2b04417f40bd2bc84c7",
```

Comment:
@lwyszomi Was this just for testing? Should probably be changed back to loc.location_id